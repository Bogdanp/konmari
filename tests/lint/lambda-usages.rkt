#lang racket/base

;; This currently works only if the lambda comes after the definition.
;; We need a way to punt on bindings.

(lambda ()
  (show "hello"))

(define (show s)
  (displayln s))
