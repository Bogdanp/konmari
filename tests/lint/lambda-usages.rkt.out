/lambda-usages.rkt:9:10:warning:identifier 'show' is never used
